### PR TITLE
Create database structure from diagram

### PR DESCRIPTION
**Pull request by fezzopro**
## PR Details

his pull request contains: 

- Create a file named `schema_based_on_diagram.sql` where you implement the database from the [diagram](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/images/clinic_diagram.png).
- Join tables from many-to-many relationships might not appear in the diagram, but you still need them.
- Add the FK indexes.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  FOREIGN KEY (invoice_id) REFERENCES invoices(id), 
+  FOREIGN KEY (treatment_id) REFERENCES treatments(id) 
+);
+
+-- Create join table
+CREATE TABLE medical_history_treatments (
+  medical_history_id INT, 
+  treatment_id INT, 
+  FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id), 
+  FOREIGN KEY (treatment_id) REFERENCES treatments(id)
```
Comment (gemmen29) [inline schema_based_on_diagram.sql:51]: - [ ] Kindly create indexes for foreign keys in the `medical_history_treatments` table as per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/database_from_diagram.md#:~:text=Remember%20to%20add%20the%20FK%20indexes.).

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (fezzopro) [reply schema_based_on_diagram.sql:51]: Thank you for your keen observation @gemmen29 
We have added 2 missing indexes. 👍

**Review by gemmen29 — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**